cmd/runtime_options.go

```diff
@@ -82,5 +91,11 @@ func getRuntimeOptions(flags *pflag.FlagSet) (lib.RuntimeOptions, error) {
 		opts.Env[k] = v
 	}
 
+	// Fallback to env
+	compatMode := opts.Env["K6_COMPATIBILITY_MODE"]
+	if !opts.CompatibilityMode.Valid && compatMode != "" {
+		opts.CompatibilityMode = null.StringFrom(compatMode)
+	}
```

Comment:
More custom fallback handling... :-/ Let me know if there's a cleaner way of doing this.

This, and the repeated flag definitions should be addressed in #883.